.circleci/config.yml

```diff
@@ -1,20 +1,54 @@
 version: 2.1
 
+orbs:
+  win: circleci/windows@1.0.0
+
 workflows:
   version: 2
   test:
     jobs:
-      - test-netcore-2-1
+      - test-netcore:
+          name: .NET Core 2.1
+          docker-image: mcr.microsoft.com/dotnet/core/sdk:2.1-focal
+          test-target-framework: netcoreapp2.1
+      - test-netcore:
+          name: .NET Core 3.1
+          docker-image: mcr.microsoft.com/dotnet/core/sdk:3.1-focal
+          test-target-framework: netcoreapp3.1
+      - test-netcore:
+          name: .NET 5.0
+          docker-image: mcr.microsoft.com/dotnet/sdk:5.0-focal
+          test-target-framework: net5.0
+      - test-windows-netframework-4-6-1:
+          name: .NET Framework 4.6.1
```

Comment:
The lower bound of our .NET Framework compatibility for this library is actually 4.5.2. The problem is that we do need an embedded HTTP server for some of our tests; WireMock.Net theoretically works in 4.5.2, but in practice it's been pretty unreliable, and EmbedIO isn't available for 4.5.2.